tox.ini

```diff
@@ -19,6 +19,8 @@ deps =
     .[testing]
     attrs18: attrs~=18.2
     attrs19: attrs~=19.1
+    attrs20: attrs~=20.3
+    attrs21: attrs~=21.3
```

Comment:
doesn't this allow for version `21.4` which is the undesired one?

Reply:
You're right, I've fixed it.